app/src/androidTest/java/edu/uc/clemenks/keepingupwithschool/ExampleInstrumentedTest.kt

```diff
@@ -15,8 +15,7 @@ import org.junit.Assert.*
  */
 @RunWith(AndroidJUnit4::class)
 class ExampleInstrumentedTest {
-    @Test
-    fun useAppContext() {
+    @Test fun useAppContext() {
```

Comment:
changed placement of "@Test" to be in same line as the function according to:
https://kotlinlang.org/docs/reference/coding-conventions.html